Application/src/Main.java

```diff
@@ -10,15 +10,15 @@
 import javafx.stage.Stage;
 import javafx.scene.image.Image;
 
-import net.Net;
-import visuals.DiagramView;
-import visuals.ExecutionLog;
+import controller.network.Net;
+import view.DiagramView;
+import view.ExecutionLog;
 import model.Menu;
-import visuals.handlers.Resizer;
+import view.handlers.Resizer;
```

Comment:
Please order the imports, like Model first, view after, controller last.
(only in this file, the other files look fine)